apis/v1/shared_types.go

```diff
@@ -334,9 +334,11 @@ const (
 	RouteReasonNoMatchingListenerHostname RouteConditionReason = "NoMatchingListenerHostname"
 
 	// This reason is used with the "Accepted" condition when there are
-	// no matching Parents. In the case of Gateways, this can occur when
-	// a Route ParentRef specifies a Port and/or SectionName that does not
-	// match any Listeners in the Gateway.
+	// no matching Parents. In the case of Gateways, this can occur when:
+	// * the route ParentRefs field is empty
+	// * no referenced parent exists
```

Comment:
```suggestion
	// * the referenced parent does not exist
```

Reply:
`parentRefs` is a slice of parents, so "no referenced parent exists" means that "none of the referenced parents exists", that is different than "the referenced parent does not exist", where we assume there is only one parent.

Reply:
Ah, yes. I forgot about this being a slice. Thanks!